lib/ansible/modules/network/f5/bigip_software_update.py

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017 F5 Networks Inc.
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: bigip_software_update
+short_description: Manage the software update settings of a BIG-IP
+description:
+  - Manage the software update settings of a BIG-IP.
+version_added: "2.5"
+options:
+  auto_check:
+    description:
+      - Specifies whether to automatically check for updates on the F5
+        Networks downloads server.
+    choices:
+      - yes
+      - no
+  auto_phone_home:
+    description:
+      - Specifies whether to automatically send phone home data to the
+        F5 Networks PhoneHome server.
+    choices:
+      - yes
+      - no
+  frequency:
+    description:
+      - Specifies the schedule for the automatic update check.
+    choices:
+      - daily
+      - monthly
+      - weekly
+notes:
+  - Requires the f5-sdk Python package on the host This is as easy as pip
```

Comment:
`C(pip install f5-sdk)`

Reply:
should i change this across all of the modules?

Reply:
It's a minor issue. Wonder if it can go in the docs fragment

Reply:
hmm, thats a good point. i havent been much concerned with optimizations lately, but that would surely be one.

Notes in doc fragments are presumably merged with notes in modules?